_rules/table-header-cells-refer-to-data-cells-d0f69e.md

```diff
@@ -0,0 +1,244 @@
+---
+id: d0f69e
+name: All table header cells have corresponding data cells
+rule_type: atomic
+description: |
+  This rule checks that each table header has corresponding data cells in a table element.
+accessibility_requirements:
+  wcag20:1.3.1: # Info and Relationships (A)
+    forConformance: true
+    failed: not satisfied
+    passed: further testing needed
+input_aspects:
+- DOM Tree
+authors:
+- Jey Nandakumar
+- Audrey Maniez
+---
+
+## Applicability
+
+This rule applies to `th` elements within a `table` that is [visible](#visible) and [included in the accessibility tree](#included-in-the-accessibility-tree).
+
+## Expectation
+
+Each target element is either the row header or the column header for data cells that are not empty (""), within the same `table`.
```

Comment:
I think the empty clause should be dropped. I would not fail example 2 under 1.3.1 (because there is no information, structure or relationships that are conveyed by presentation that cannot be programmatically determined).